data/Dockerfile.template

```diff
@@ -1,4 +1,4 @@
-FROM resin/%%RESIN_MACHINE_NAME%%-alpine-node:8.9.4-slim
+FROM resin/%%RESIN_MACHINE_NAME%%-node:7-slim
```

Comment:
How come the move to debian?

Reply:
@CameronDiver There's an issue with Alpine, we get `could not set hostname` permission errors, which *sometimes* causes the RPi3 to fail, and *always* causes the IOT2040 to fail. I've not had time to track them down yet, I'd love to move back to Alpine but just don't have time atm. :-/